ConwaysGameOfLife.Tests/SourceData.cs

```diff
@@ -0,0 +1,16 @@
+namespace ConwaysGameOfLife.Tests
+{
+    public static class SourceData
+    {
+        public static Universe UniverseInitializedAllDeadCells()
+        {
+            var initData = new CellState[][]
+            {
+                new []  {CellState.Dead,   CellState.Dead,    CellState.Dead },
+                new []  {CellState.Dead,   CellState.Dead,    CellState.Dead },
+                new []  {CellState.Dead,   CellState.Dead,    CellState.Dead }
+            };
+            return new Universe(initData);
+        }
+    }

```

Comment:
perhaps the source data can be structured a different way so you don't have to create a copy constructor in the actual universe class just for the purposes of testing